docs/book/stacks-and-components/component-guide/orchestrators/skypilot-vm.md

```diff
@@ -262,7 +262,8 @@ For additional configuration of the Skypilot orchestrator, you can pass `Skypilo
 from zenml.integrations.skypilot.flavors.skypilot_orchestrator_flavor import SkypilotBaseOrchestratorSettings
 
 skypilot_settings = SkypilotBaseOrchestratorSettings(
-    instance_type="m5.large",
+**Code Example:**
+
     cpus="2",
     memory="16",
     accelerators="V100:2",
```

Comment:
Uhm here something got messed up, the **Code Example:** somehow got inserted in the middle of the code block 😅